_data/faq.yml

```diff
@@ -1,3 +1,131 @@
+                TBC  
+      - id: design-font
+        en:
+            title: Why are all texts now bigger?
+            content: |
+                We do now follow the Android style guidelines for font sizes. We also had many users telling us, that the texts rather small and hard to read compared to other apps on the same device.
+                
+                Give it a try for some days...its probably just a matter of getting used to the new look.
+                Besides that, you can now also adjust the font size in c:geo by adjusting your system settings accordingly as c:geo is now following that system setting.                
+        de:
+            title: Warum sind jetzt alle Texte größer?
+            content: |
+                TBC 
+      - id: design-popup
+        en:
+            title: The cache name is missing on the map!
+            content: |
+                The cache popup on the map is now showing the geocode in its title instead of the cache name, but the cache name is shown immediately below.
+                
+                We changed this because long cache names have always be truncated in the past due to the limited space on the title bar. Now you can see the whole cache name instead of only a part of it.
+        de:
+            title: Der Cache-Name wird auf der Karte nicht mehr angezeigt!
+            content: |
+                TBC  
+- id: filter
+  title:
+      en: New filtering framework
+      de: Neue Filter-Methoden
+  items:
+      - id: filter-generic
+        en:
+            title: Why did you change the filtering?
+            content: |
+                There have been several different filtering methods included into c:geo over the last years. As they have been developed independently of one another, there have been
+                several limitations and issues with that, which we could not fix.
+                
+                Instead of trying to glue those existing filters together, we therefore decided to make a clean, new and more powerful approach.
+        de:
+            title: Warum habt ihr die Filterung geändert?
+            content: |
+                TBC  
+      - id: filter-changes
+        en:
+            title: What has changed for filtering?
+            content: |             
+                There used to be several different filters at different places in c:geo, which did only partly interact properly:
+                - The global hiding of found, owned, disabled, archived caches
+                - The global switch to filter on a single cache type within the whole app
+                - A bunch of filters for your stored caches (e.g. for certain states, attributes, etc.)
+                
+                We basically removed all of those filters and replaced them by a common filter framework with more and more flexible filtering options.
+                
+                There are now two independent filter contexts with identical filtering options:
```

Comment:
"There are now two independent filter contexts stored in the app (with identical filtering options):"